src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-pypemicro.py

```diff
@@ -0,0 +1,39 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2022 PyInstaller Development Team.
+#
+# This file is distributed under the terms of the GNU General Public
+# License (version 2.0 or later).
+#
+# The full license is available in LICENSE.GPL.txt, distributed with
+# this software.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+# ------------------------------------------------------------------
+
+# Hook for the pypemicro module: https://github.com/nxpmicro/pypemicro
+
+import os
+from PyInstaller.utils.hooks import remove_prefix, get_package_paths, is_module_satisfies
+from PyInstaller.log import logger
+from PyInstaller.compat import is_darwin
+
+binaries = list()
+if is_module_satisfies('pyinstaller < 5.0'):
+    logger.warning("hook-pypemicro requires pyinstaller >= 5.0")
+else:
+    from PyInstaller import isolated
+
+    @isolated.decorate
+    def get_safe_libs():
+        from pypemicro import PyPemicro
+        libs = PyPemicro.get_pemicro_lib_list()
+        return libs
+
+    pkg_base, pkg_dir = get_package_paths("pypemicro")
+    for lib in get_safe_libs():
+        source_path = lib['path']
+        source_name = lib['name']
+        dest = remove_prefix(source_path, os.path.dirname(pkg_base) + os.sep)
+        binaries.append((os.path.join(source_path, source_name), dest))
+        if is_darwin:
+            binaries.append((os.path.join(source_path, 'libusb.dylib'), dest))
```

Comment:
We should probably check that `os.path.join(source_path, 'libusb.dylib')` exists before trying to add it.